core/src/main/scala/akka/persistence/r2dbc/state/scaladsl/R2dbcDurableStateStore.scala

```diff
@@ -120,4 +126,45 @@ class R2dbcDurableStateStore[A](system: ExtendedActorSystem, config: Config, cfg
       offset: Offset): Source[DurableStateChange[A], NotUsed] =
     bySlice.liveBySlices("changesBySlices", entityTypeHint, minSlice, maxSlice, offset)
 
+  override def currentPersistenceIds(afterId: Option[String], limit: Long): Source[String, NotUsed] =
+    stateDao.persistenceIds(afterId, limit)
+
+  def currentPersistenceIds(): Source[String, NotUsed] = {
```

Comment:
Isn't the no-parameter one only for the event journal?

Reply:
Yes, but I'm adding both for completeness. `currentPersistenceIds()` is for fetching all and `currentPersistenceIds(afterId)` is for paging.